packages/ember-data/lib/system/relationships/state/belongs-to.js

```diff
@@ -110,6 +110,15 @@ BelongsToRelationship.prototype.findRecord = function() {
   }
 };
 
+BelongsToRelationship.prototype._super$findLink = Relationship.prototype.findLink;
+BelongsToRelationship.prototype.findLink = function() {
```

Comment:
You should probably be using https://github.com/csantero/data/blob/dont-fetch/packages/ember-data/lib/system/relationships/state/relationship.js#L18 instead of the find link, because that tells us whether we already loaded data for the relationship. Can you put this on the super class and thus make it work for many relationships as well?